components/brave_news/browser/brave_news_pref_manager_unittest.cc

```diff
@@ -4,9 +4,11 @@
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
 #include "brave/components/brave_news/browser/brave_news_pref_manager.h"
+
 #include "base/containers/contains.h"
 #include "base/containers/flat_set.h"
-#include "brave/components/brave_news/common/brave_news.mojom.h"
+#include "brave/components/brave_news/common/brave_news.mojom-shared.h"
```

Comment:
Just double checking,  do we want to include .mojom-shared.h instead of mojom.h?

Reply:
Whoops, we should not!